sphinx_rtd_theme/versions.html

```diff
@@ -1,34 +1,87 @@
-{% if READTHEDOCS %}
-{# Add rst-badge after rst-versions for small badge style. #}
-  <div class="rst-versions" data-toggle="rst-versions" role="note" aria-label="{{ _('Versions') }}">
+<script type="text/javascript">
+ document.addEventListener("readthedocs-addons-data-ready", function(event) {
+     const config = event.detail.data();
+
+     const downloadsNameDisplay = {
+      pdf: "PDF",
+      epub: "EPUB",
+      htmlzip: "HTML",
+    };
+
+     // TODO: remove section if there is no content.
+     // TODO: add `rtd-current-item` to the selected one.
+     const flyout = `
+  <div class="rst-versions" data-toggle="rst-versions" role="note">
     <span class="rst-current-version" data-toggle="rst-current-version">
       <span class="fa fa-book"> Read the Docs</span>
-      v: {{ current_version }}
+      v: ${ config.versions.current.slug }
       <span class="fa fa-caret-down"></span>
     </span>
     <div class="rst-other-versions">
-      <dl>
-        <dt>{{ _('Versions') }}</dt>
-        {% for slug, url in versions %}
-          <dd><a href="{{ url }}">{{ slug }}</a></dd>
-        {% endfor %}
-      </dl>
-      <dl>
-        <dt>{{ _('Downloads') }}</dt>
-        {% for type, url in downloads %}
-          <dd><a href="{{ url }}">{{ type }}</a></dd>
-        {% endfor %}
-      </dl>
-      <dl>
-        {# Translators: The phrase "Read the Docs" is not translated #}
-        <dt>{{ _('On Read the Docs') }}</dt>
+      <div class="injected">
+        <dl>
+          <dt>{{ _('Languages') }}</dt>
+          ${ config.projects.translations.map(
+            (translation) => `
+            <dd><a href="${ translation.urls.documentation }">${ translation.slug }</a></dd>
+          `).join("\n")}
+        </dl>
```

Comment:
You probably want conditionals around each of these sections. You can use Lit directives if you want to bring in Lit, or you can do this in JS with string appending or breaking this up into multiple functions/methods/etc.

Reply:
Yes. I need the conditionals to avoid showing the sections when there is no content.

I don't want to introduce Lit in this PR, but it's something I want to discuss more and consider in the following iteration. Related to my previous comment, https://github.com/readthedocs/sphinx_rtd_theme/pull/1526#discussion_r1568648856

Reply:
I did this in ef0a57e8526d032534356b74862dcc34012f7c03